### DISCUSSION TITLE
não consigo acessar a aba e configurações

### DISCUSSION THREAD

**Discussion by AleeJr97** [alyssapiresfernandescefet/openrpg#6] [Q&A] [answered]
então, eu estou utilizando a ficha com meus amigos em um rpg, só que no painel do mestre quando e tento acessar a aba de configurações aparece uma tela de erro e não consigo usar, o resto está funcionando normalmente.

![unknown](https://user-images.githubusercontent.com/101086293/162684419-fe4da0ed-98e4-46d0-8944-2e84272d6588.png)

**Comment by lucaspiresfernandes** [comment, marked as answer]
Olá! Isso provavelmente aconteceu porque você atualizou o Open RPG para a última versão, que não é compatível com versões anteriores. Me manda uma mensagem no discord: That Alyssa#0615. Lá eu te ajudo a resolver esse problema.
